File: Python/1168.py
casos = int(input())
for i in range(0,casos):
    leds = 0
    entrada = input()
    for i in range(0, len(entrada)):
        if entrada[i] == '1':
            leds += 2
        if entrada[i] == '2':
            leds += 5
        if entrada[i] == '3':
            leds += 5
        if entrada[i] == '4':
            leds += 4
        if entrada[i] == '5':
            leds += 5
        if entrada[i] == '6':
            leds += 6
        if entrada[i] == '7':
            leds += 3
        if entrada[i] == '8':
            leds += 7
        if entrada[i] == '9':
            leds += 6
        if entrada[i] == '0':
            leds += 6
    print("{} leds".format(int(leds)))

File: golang/1168/1168.go
package main
import "fmt"
func main(){
	var casos int
	fmt.Scanln(&casos)
	for i:= 0; i < casos; i++ {
		var entrada string
		fmt.Scanln(&entrada)
		aux := 0
		soma := 0
		for true{
			if len(entrada) == aux{
				break
			}
			if entrada[aux] == '1'{
				soma = soma + 2
			}
			if entrada[aux] == '2'{
				soma = soma + 5
			}
			if entrada[aux] == '3'{
				soma = soma + 5
			}
			if entrada[aux] == '4'{
				soma = soma + 4
			}
			if entrada[aux] == '5'{
				soma = soma + 5
			}
			if entrada[aux] == '6'{
				soma = soma + 6
			}
			if entrada[aux] == '7'{
				soma = soma + 3
			}
			if entrada[aux] == '8'{
				soma = soma + 7
			}
			if entrada[aux] == '9'{
				soma = soma + 6
			}
			if entrada[aux] == '0'{
				soma = soma + 6
			}
			aux = aux + 1;
		}
		fmt.Printf("%d leds\n",soma)
	}
}